src/Widgets/FirmwareHeaderRow.vala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA
+ *
+ * Authored by: Marius Meisenzahl <mariusmeisenzahl@gmail.com>
+ */
+
+public class About.Widgets.FirmwareHeaderRow : Gtk.Label {
+    construct {
+        xalign = 0;
+        margin = 3;
+        get_style_context ().add_class (Granite.STYLE_CLASS_H4_LABEL);
+    }
+
+    public FirmwareHeaderRow.updatable (uint num_updates) {
+        Object (
+            label: ngettext ("%u Update Available", "%u Updates Available", num_updates).printf (num_updates)
+        );
+    }
+
+    public FirmwareHeaderRow.up_to_date () {
+        Object (label: _("Up to Date"));
+    }
```

Comment:
This seems kind of overkill when it's pretty easy to do:
```
var updateable_header = new FirmwareHeaderRow (
    ngettext ("%u Update Available", "%u Updates Available", num_updates).printf (num_updates)
);
```
and
```
var updated_header = new FirmwareHeaderRow (_("Up to Date"));
```

Since this is such a simple class, it can probably just be private to the FirmwareView without these additional constructors